### DISCUSSION TITLE
v380pro Telnet не подключается

### DISCUSSION THREAD

**Discussion by MrDogYar** [Arkady23/V380-Web#70] [General]
Добрый день!
Камера v380pro (в сети по MAC определилась как AltoBeam) прошивка Hw_AWT3610E_XHR_E_V1.0_WF_2023114
Скинул в корень архив.
Вставил в камеру.
Включил.
Подождал 5 минут.
Telnet не коннектится.
Networc error. Connection refuse

**Comment by nvrskRU** [comment]
разночтения начинаются с момента формата карты:
инструкция гласит, что делать надо FAT32, а когда камера включается, и открываешь её в телефоне приложением - ругается, что, мол, карта не отформатирована.
форматируешь карту, и она внезапно становится ExFAT.

при этом попытки записать на отформатированную таким образом карту файлов с "мануальной установкой" ни к чему не привели - скрипты не исполняются, порты не открываются, телнет недоступен...

**Comment by Arkady23** [comment]
Что такое "telnet не коннектится"? Вы в новой камере сначала проверьте есть ли доступ через telnet. Если доступа через telnet нет... т.е. китайцы его специально заблокировали, чтобы разные умники не лазили, то что-то установить не получится в принципе. Всё идет, всё изменяется. Сейчас V380 это видимо не то, что было раньше.

**Comment by NaivE2k4** [comment]
@Arkady23 Добрый день!
Подскажите пожалуйста, у меня тоже камера A6 V380 pro, и я тоже ищу способ открыть в ней доп функционал.
Из описания Вашего решения я предполагал, что загрузившись с флешки, на камере откроется телнет и дальше можно запустить через него сервер.
Однако, телнет закрыт, NMAP вообще показывает только 8800 открытый. Я верно понимаю, что это тупик, и Ваше решение тут не поможет?

**Reply by Arkady23** [reply]
Да, без telnet-а никак. Когда telnet есть, можно что-то доработать.

**Reply by NaivE2k4** [reply]
> Да, без telnet-а никак. Когда telnet есть, можно что-то доработать.

@Arkady23 но, судя по форумам, что я прочитал - существуют способы открытия телнета подкладыванием на sd-карточку специальных ini-файликов, верно?
Ещё подскажите пожалуйста, влияют ли на результат encoding и line ending в конфигах? Не увидел этого ни в одном сообщении на 4pda, но предполагаю, что люди которые предлагают решения, вроде Вас, уважаемый Аркадий - сидят на Линуксе, а ищущие решений - сидят на Винде, а там иные эндинги и из-за этого в теории скрипты и конфиги могут не читаться.
Верно в понимаю что нужно ANSI+ UNIX (LF)?

**Comment by Arkady23** [comment]
Я люблю Windows. Я на Windows. Если вы про CR и LF — конец строки у текстовых файлов Windows с кодами символов 13 и 10. То да, есть разница. Обратите внимание, когда будете править текстовые файлы. У линукса конец строки просто LF - код символа 10. Мне кажется, что подключение telnet через помещение на sd специального тестового ini файла уже давно не работает. Будет работать, если у вас очень старая камера, но от старых камер известен пароль root, поэтому этот ini файл уже не имеет никакого значения.

**Comment by Arkady23** [comment]
Появился новый способ установки веб на камеру. В ближайшее время опишу на главной странице.
